mcs/class/referencesource/mscorlib/system/security/cryptography/rc2cryptoserviceprovider.cs

```diff
@@ -63,7 +63,12 @@ public sealed class RC2CryptoServiceProvider : RC2 {
                 return m_use40bitSalt;
             }
             set {
+#if MONO
```

Comment:
Ah ok that makes sense. What should we do about the CommonCrypto implementation? Do the same there (which would require a bit more generator.cs surgery)?